conda-forge CI build failure on Linux mid September 2023 against abseil 20230802 (transitive dependency via protobuf)

## Environment
* OS Version: conda-forge (on top of Ubuntu 22.04)
* Source or binary build?
  Source

## Description

Build against with : 

~~~
2023-09-14T02:54:40.6108443Z [650/1197] Building CXX object gazebo/physics/CMakeFiles/gazebo_physics.dir/simbody/SimbodyPhysics.cc.o
2023-09-14T02:54:40.6110158Z FAILED: gazebo/physics/CMakeFiles/gazebo_physics.dir/simbody/SimbodyPhysics.cc.o 
2023-09-14T02:54:40.6119227Z /usr/share/miniconda3/envs/test/bin/x86_64-conda-linux-gnu-c++ -DBOOST_ATOMIC_DYN_LINK -DBOOST_ATOMIC_NO_LIB -DBOOST_DATE_TIME_DYN_LINK -DBOOST_DATE_TIME_NO_LIB -DBOOST_FILESYSTEM_DYN_LINK -DBOOST_FILESYSTEM_NO_LIB -DBOOST_IOSTREAMS_DYN_LINK -DBOOST_IOSTREAMS_NO_LIB -DBOOST_PROGRAM_OPTIONS_DYN_LINK -DBOOST_PROGRAM_OPTIONS_NO_LIB -DBOOST_REGEX_DYN_LINK -DBOOST_REGEX_NO_LIB -DBOOST_SYSTEM_DYN_LINK -DBOOST_SYSTEM_NO_LIB -DBOOST_THREAD_DYN_LINK -DBOOST_THREAD_NO_LIB -DBUILDING_DLL -DBUILDING_DLL_GZ_PHYSICS -DDART_ACTIVE_LOG_LEVEL=2 -DDART_HAVE_spdlog=1 -DFMT_HEADER_ONLY=1 -DFMT_SHARED -DHAVE_OPENGL -DIGN_PROFILER_ENABLE=0 -DLIBBULLET_VERSION=3.24 -DLIBBULLET_VERSION_GT_282 -DPROTOBUF_USE_DLLS -DSPDLOG_FMT_EXTERNAL -DTINYXML2_IMPORT -DdIDEDOUBLE -Dgazebo_physics_EXPORTS -I/home/runner/work/gazebo-classic/gazebo-classic/test/gtest/include -I/home/runner/work/gazebo-classic/gazebo-classic -I/home/runner/work/gazebo-classic/gazebo-classic/build -I/usr/share/miniconda3/envs/test/include/libusb-1.0 -isystem /home/runner/work/gazebo-classic/gazebo-classic/deps/opende/include -isystem /usr/share/miniconda3/envs/test/include/ignition/msgs5 -isystem /usr/share/miniconda3/envs/test/include/ignition/math6 -isystem /usr/share/miniconda3/envs/test/include/ignition/transport8 -isystem /usr/share/miniconda3/envs/test/include/ignition/common3 -isystem /usr/share/miniconda3/envs/test/include/ignition/fuel_tools4 -isystem /usr/share/miniconda3/envs/test/include/OGRE/RTShaderSystem -isystem /usr/share/miniconda3/envs/test/include/OGRE -isystem /usr/share/miniconda3/envs/test/include/OGRE/Terrain -isystem /usr/share/miniconda3/envs/test/include/OGRE/Paging -isystem /usr/share/miniconda3/envs/test/include/OGRE/Overlay -isystem /usr/share/miniconda3/envs/test/include/sdformat-9.8 -isystem /usr/share/miniconda3/envs/test/include/bullet -isystem /usr/share/miniconda3/envs/test/include/simbody -isystem /usr/share/miniconda3/envs/test/include/uuid -isystem /usr/share/miniconda3/envs/test/include/sdformat-9.8/sdf/.. -isystem /usr/share/miniconda3/envs/test/include/ignition/cmake2 -isystem /usr/share/miniconda3/envs/test/include/eigen3 -fvisibility-inlines-hidden -fmessage-length=0 -march=nocona -mtune=haswell -ftree-vectorize -fPIC -fstack-protector-strong -fno-plt -O2 -ffunction-sections -pipe -isystem /usr/share/miniconda3/envs/test/include       -Wall -Wextra -Wno-long-long -Wno-unused-value -Wfloat-equal -Wshadow -Wswitch-default -Wmissing-include-dirs -pedantic -fvisibility=hidden -fvisibility-inlines-hidden   -s -O3 -DNDEBUG -mssse3 -msse3 -mfpmath=sse -msse -msse2  -fPIC   -fPIC -I/usr/share/miniconda3/envs/test/include/bullet -I/usr/share/miniconda3/envs/test/include -I"/usr/share/miniconda3/envs/test/include/simbody" -I/usr/share/miniconda3/envs/test/include/uuid -mfpmath=sse -msse -msse2 -msse3 -mssse3 -MD -MT gazebo/physics/CMakeFiles/gazebo_physics.dir/simbody/SimbodyPhysics.cc.o -MF gazebo/physics/CMakeFiles/gazebo_physics.dir/simbody/SimbodyPhysics.cc.o.d -o gazebo/physics/CMakeFiles/gazebo_physics.dir/simbody/SimbodyPhysics.cc.o -c /home/runner/work/gazebo-classic/gazebo-classic/gazebo/physics/simbody/SimbodyPhysics.cc
2023-09-14T02:54:40.6124466Z In file included from /usr/share/miniconda3/envs/test/include/absl/strings/str_format.h:78,
2023-09-14T02:54:40.6124977Z                  from /usr/share/miniconda3/envs/test/include/absl/crc/crc32c.h:32,
2023-09-14T02:54:40.6125465Z                  from /usr/share/miniconda3/envs/test/include/absl/crc/internal/crc_cord_state.h:23,
2023-09-14T02:54:40.6126176Z                  from /usr/share/miniconda3/envs/test/include/absl/strings/cord.h:79,
2023-09-14T02:54:40.6126687Z                  from /usr/share/miniconda3/envs/test/include/google/protobuf/io/coded_stream.h:134,
2023-09-14T02:54:40.6127392Z                  from /home/runner/work/gazebo-classic/gazebo-classic/build/gazebo/msgs/poses_stamped.pb.h:24,
2023-09-14T02:54:40.6128079Z                  from /home/runner/work/gazebo-classic/gazebo-classic/gazebo/physics/PhysicsTypes.hh:26,
2023-09-14T02:54:40.6128730Z                  from /home/runner/work/gazebo-classic/gazebo-classic/gazebo/physics/Model.hh:28,
2023-09-14T02:54:40.6129396Z                  from /home/runner/work/gazebo-classic/gazebo-classic/gazebo/physics/simbody/SimbodyModel.hh:21,
2023-09-14T02:54:40.6130098Z                  from /home/runner/work/gazebo-classic/gazebo-classic/gazebo/physics/simbody/SimbodyPhysics.cc:23:
2023-09-14T02:54:40.6131506Z /usr/share/miniconda3/envs/test/include/absl/strings/internal/str_format/arg.h: In instantiation of 'struct absl::lts_20230802::str_format_internal::FormatArgImpl::store_by_value<const unsigned int&>':
2023-09-14T02:54:40.6133116Z /usr/share/miniconda3/envs/test/include/absl/strings/internal/str_format/arg.h:435:59:   required from 'struct absl::lts_20230802::str_format_internal::FormatArgImpl::storage_policy<const unsigned int&>'
2023-09-14T02:54:40.6134117Z /usr/share/miniconda3/envs/test/include/absl/strings/internal/str_format/arg.h:472:64:   required from 'absl::lts_20230802::str_format_internal::FormatArgImpl::FormatArgImpl(const T&) [with T = unsigned int]'
2023-09-14T02:54:40.6135439Z /usr/share/miniconda3/envs/test/include/absl/strings/str_format.h:395:29:   required from 'absl::lts_20230802::str_format_internal::Streamable absl::lts_20230802::StreamFormat(FormatSpec<Args ...>&, const Args& ...) [with Args = {unsigned int}; FormatSpec<Args ...> = str_format_internal::FormatSpecTemplate<(absl::lts_20230802::FormatConversionCharSet)655355>]'
2023-09-14T02:54:40.6136204Z /usr/share/miniconda3/envs/test/include/absl/crc/crc32c.h:184:34:   required from here
2023-09-14T02:54:40.6137339Z /usr/share/miniconda3/envs/test/include/absl/strings/internal/str_format/arg.h:423:49: error: call to non-'constexpr' function 'bool SimTK::operator<=(const L&, const R&) [with L = long unsigned int; R = absl::lts_20230802::str_format_internal::FormatArgImpl::<unnamed enum>]'
2023-09-14T02:54:40.6138004Z   423 |       : std::integral_constant<bool, (sizeof(T) <= kInlinedSpace) &&
2023-09-14T02:54:40.6138331Z       |                                      ~~~~~~~~~~~^~~~~~~~~~~~~~~~~
2023-09-14T02:54:40.6138714Z In file included from /usr/share/miniconda3/envs/test/include/simbody/SimTKcommon/basics.h:38,
2023-09-14T02:54:40.6139159Z                  from /usr/share/miniconda3/envs/test/include/simbody/SimTKcommon.h:32,
2023-09-14T02:54:40.6139571Z                  from /usr/share/miniconda3/envs/test/include/simbody/Simbody.h:38,
2023-09-14T02:54:40.6140171Z                  from /home/runner/work/gazebo-classic/gazebo-classic/gazebo/physics/simbody/simbody_inc.h:22,
2023-09-14T02:54:40.6140788Z                  from /home/runner/work/gazebo-classic/gazebo-classic/gazebo/physics/simbody/SimbodyLink.hh:25,
2023-09-14T02:54:40.6141410Z                  from /home/runner/work/gazebo-classic/gazebo-classic/gazebo/physics/simbody/SimbodyPhysics.cc:24:
2023-09-14T02:54:40.6142400Z /usr/share/miniconda3/envs/test/include/simbody/SimTKcommon/internal/common.h:653:6: note: 'bool SimTK::operator<=(const L&, const R&) [with L = long unsigned int; R = absl::lts_20230802::str_format_internal::FormatArgImpl::<unnamed enum>]' declared here
2023-09-14T02:54:40.6142983Z   653 | bool operator<=(const L& left, const R& right)
2023-09-14T02:54:40.6143258Z       |      ^~~~~~~~
2023-09-14T02:54:40.6143785Z /usr/share/miniconda3/envs/test/include/absl/strings/internal/str_format/arg.h:423:67: note: in template argument for type 'bool'
2023-09-14T02:54:40.6144248Z   423 |       : std::integral_constant<bool, (sizeof(T) <= kInlinedSpace) &&
2023-09-14T02:54:40.6144709Z       |                                      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~
2023-09-14T02:54:40.6145023Z   424 |                                          (std::is_integral<T>::value ||
2023-09-14T02:54:40.6145335Z       |                                          ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
2023-09-14T02:54:40.6145640Z   425 |                                           std::is_floating_point<T>::value ||
2023-09-14T02:54:40.6145959Z       |                                           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
2023-09-14T02:54:40.6147444Z   426 |                                           std::is_pointer<T>::value ||
2023-09-14T02:54:40.6147754Z       |                                           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
2023-09-14T02:54:40.6148262Z   427 |                                           std::is_same<VoidPtr, T>::value)> {};
2023-09-14T02:54:40.6148584Z       |                                           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
2023-09-14T02:54:40.6149405Z /usr/share/miniconda3/envs/test/include/absl/strings/internal/str_format/arg.h: In instantiation of 'struct absl::lts_20230802::str_format_internal::FormatArgImpl::storage_policy<const unsigned int&>':
2023-09-14T02:54:40.6150404Z /usr/share/miniconda3/envs/test/include/absl/strings/internal/str_format/arg.h:472:64:   required from 'absl::lts_20230802::str_format_internal::FormatArgImpl::FormatArgImpl(const T&) [with T = unsigned int]'
2023-09-14T02:54:40.6151713Z /usr/share/miniconda3/envs/test/include/absl/strings/str_format.h:395:29:   required from 'absl::lts_20230802::str_format_internal::Streamable absl::lts_20230802::StreamFormat(FormatSpec<Args ...>&, const Args& ...) [with Args = {unsigned int}; FormatSpec<Args ...> = str_format_internal::FormatSpecTemplate<(absl::lts_20230802::FormatConversionCharSet)655355>]'
2023-09-14T02:54:40.6152462Z /usr/share/miniconda3/envs/test/include/absl/crc/crc32c.h:184:34:   required from here
2023-09-14T02:54:40.6153322Z /usr/share/miniconda3/envs/test/include/absl/strings/internal/str_format/arg.h:435:59: error: 'value' is not a member of 'absl::lts_20230802::str_format_internal::FormatArgImpl::store_by_value<const unsigned int&>'
2023-09-14T02:54:40.6153882Z   435 |                                     : (store_by_value<T>::value ? ByValue
2023-09-14T02:54:40.6154350Z       |                                                           ^~~~~
2023-09-14T02:54:40.6155123Z /usr/share/miniconda3/envs/test/include/absl/strings/internal/str_format/arg.h: In instantiation of 'absl::lts_20230802::str_format_internal::FormatArgImpl::FormatArgImpl(const T&) [with T = unsigned int]':
2023-09-14T02:54:40.6156437Z /usr/share/miniconda3/envs/test/include/absl/strings/str_format.h:395:29:   required from 'absl::lts_20230802::str_format_internal::Streamable absl::lts_20230802::StreamFormat(FormatSpec<Args ...>&, const Args& ...) [with Args = {unsigned int}; FormatSpec<Args ...> = str_format_internal::FormatSpecTemplate<(absl::lts_20230802::FormatConversionCharSet)655355>]'
2023-09-14T02:54:40.6157195Z /usr/share/miniconda3/envs/test/include/absl/crc/crc32c.h:184:34:   required from here
2023-09-14T02:54:40.6158054Z /usr/share/miniconda3/envs/test/include/absl/strings/internal/str_format/arg.h:472:64: error: 'value' is not a member of 'absl::lts_20230802::str_format_internal::FormatArgImpl::storage_policy<const unsigned int&>'
2023-09-14T02:54:40.6158629Z   472 |         std::is_same<D, const T&>::value || storage_policy<D>::value == ByValue,
2023-09-14T02:54:40.6158961Z       |                                                                ^~~~~
2023-09-14T02:54:40.9959552Z [651/1197] Building CXX object gazebo/physics/CMakeFiles/gazebo_physics.dir/simbody/SimbodyMultiRayShape.cc.o
2023-09-14T02:54:46.9945819Z [652/1197] Building CXX object gazebo/physics/CMakeFiles/gazebo_physics.dir/simbody/SimbodyPlaneShape.cc.o
2023-09-14T02:54:46.9946908Z ninja: build stopped: subcommand failed.
2023-09-14T02:54:47.0421062Z ##[error]Process completed with exit code 1.
~~~

--- a/gazebo/physics/simbody/SimbodyPhysics.cc
+++ b/gazebo/physics/simbody/SimbodyPhysics.cc
@@ -65,7 +65,6 @@ typedef boost::shared_ptr<gazebo::physics::SimbodyJoint> SimbodyJointPtr;
 
 using namespace gazebo;
 using namespace physics;
-using namespace SimTK;
 
 GZ_REGISTER_PHYSICS_ENGINE("simbody", SimbodyPhysics)
 
@@ -314,7 +313,7 @@ void SimbodyPhysics::InitModel(const physics::ModelPtr _model)
     else
     {
       //---------------------- GENERATE MULTIBODY GRAPH ------------------------
-      MultibodyGraphMaker mbgraph;
+      SimTK::MultibodyGraphMaker mbgraph;
       this->CreateMultibodyGraph(mbgraph, _model);
       // Optional: dump the graph to stdout for debugging or curiosity.
       // mbgraph.dumpGraph(gzdbg);
@@ -929,7 +928,7 @@ void SimbodyPhysics::AddStaticModelToSimbodySystem(
     if (simbodyLink)
     {
       this->AddCollisionsToLink(simbodyLink.get(), this->matter.updGround(),
-        ContactCliqueId());
+        SimTK::ContactCliqueId());
       simbodyLink->masterMobod = this->matter.updGround();
     }
     else
@@ -946,7 +945,7 @@ void SimbodyPhysics::AddDynamicModelToSimbodySystem(
   // Generate a contact clique we can put collision geometry in to prevent
   // self-collisions.
   // \TODO: put this in a gazebo::physics::SimbodyModel class
-  ContactCliqueId modelClique = ContactSurface::createNewContactClique();
+  SimTK::ContactCliqueId modelClique = SimTK::ContactSurface::createNewContactClique();
 
   // Will specify explicitly when needed
   // Record the MobilizedBody for the World link.
@@ -960,7 +959,7 @@ void SimbodyPhysics::AddDynamicModelToSimbodySystem(
     // Get a mobilizer from the graph, then extract its corresponding
     // joint and bodies. Note that these don't necessarily have equivalents
     // in the GazeboLink and GazeboJoint inputs.
-    const MultibodyGraphMaker::Mobilizer& mob = _mbgraph.getMobilizer(mobNum);
+    const SimTK::MultibodyGraphMaker::Mobilizer& mob = _mbgraph.getMobilizer(mobNum);
     const std::string& type = mob.getJointTypeName();
 
     // The inboard body always corresponds to one of the input links,
@@ -974,7 +973,7 @@ void SimbodyPhysics::AddDynamicModelToSimbodySystem(
     SimbodyLink* gzOutb =
       static_cast<SimbodyLink*>(mob.getOutboardMasterBodyRef());
 
-    const MassProperties massProps =
+    const SimTK::MassProperties massProps =
         gzOutb->GetEffectiveMassProps(mob.getNumFragments());
 
     // debug
@@ -987,9 +986,9 @@ void SimbodyPhysics::AddDynamicModelToSimbodySystem(
     //         << "\n";
 
     // This will reference the new mobilized body once we create it.
-    MobilizedBody mobod;
+    SimTK::MobilizedBody mobod;
 
-    MobilizedBody parentMobod =
+    SimTK::MobilizedBody parentMobod =
       gzInb == nullptr ? this->matter.Ground() : gzInb->masterMobod;
 
     if (mob.isAddedBaseMobilizer())
@@ -1001,9 +1000,9 @@ void SimbodyPhysics::AddDynamicModelToSimbodySystem(
       GZ_ASSERT(type == "free", "type is not 'free', not allowed.");
       if (type == "free")
       {
-        MobilizedBody::Free freeJoint(
-            parentMobod,  Transform(),
-            massProps,    Transform());
+        SimTK::MobilizedBody::Free freeJoint(
+            parentMobod,  SimTK::Transform(),
+            massProps,    SimTK::Transform());
 
         SimTK::Transform inboard_X_ML;
         if (gzInb == nullptr)
@@ -1036,26 +1035,26 @@ void SimbodyPhysics::AddDynamicModelToSimbodySystem(
       // Find inboard and outboard frames for the mobilizer; these are
       // parent and child frames or the reverse.
 
-      const Transform& X_IF0 = isReversed ? gzJoint->xCB : gzJoint->xPA;
-      const Transform& X_OM0 = isReversed ? gzJoint->xPA : gzJoint->xCB;
+      const SimTK::Transform& X_IF0 = isReversed ? gzJoint->xCB : gzJoint->xPA;
+      const SimTK::Transform& X_OM0 = isReversed ? gzJoint->xPA : gzJoint->xCB;
 
-      const MobilizedBody::Direction direction =
-          isReversed ? MobilizedBody::Reverse : MobilizedBody::Forward;
+      const SimTK::MobilizedBody::Direction direction =
+          isReversed ? SimTK::MobilizedBody::Reverse : SimTK::MobilizedBody::Forward;
 
       if (type == "free")
       {
-        MobilizedBody::Free freeJoint(
+        SimTK::MobilizedBody::Free freeJoint(
             parentMobod,  X_IF0,
             massProps,          X_OM0,
             direction);
-        Transform defX_FM = isReversed ? Transform(~gzJoint->defxAB)
+        SimTK::Transform defX_FM = isReversed ? SimTK::Transform(~gzJoint->defxAB)
                                        : gzJoint->defxAB;
         freeJoint.setDefaultTransform(defX_FM);
         mobod = freeJoint;
       }
       else if (type == "screw")
       {
-        UnitVec3 axis(
+        SimTK::UnitVec3 axis(
           SimbodyPhysics::Vector3ToVec3(
             gzJoint->AxisFrameOffset(0).RotateVector(
             gzJoint->LocalAxis(0))));
@@ -1071,10 +1070,10 @@ void SimbodyPhysics::AddDynamicModelToSimbodySystem(
         }
 
         // Simbody's screw joint axis (both rotation and translation) is along Z
-        Rotation R_JZ(axis, ZAxis);
-        Transform X_IF(X_IF0.R()*R_JZ, X_IF0.p());
-        Transform X_OM(X_OM0.R()*R_JZ, X_OM0.p());
-        MobilizedBody::Screw screwJoint(
+        SimTK::Rotation R_JZ(axis, SimTK::ZAxis);
+        SimTK::Transform X_IF(X_IF0.R()*R_JZ, X_IF0.p());
+        SimTK::Transform X_OM(X_OM0.R()*R_JZ, X_OM0.p());
+        SimTK::MobilizedBody::Screw screwJoint(
             parentMobod,      X_IF,
             massProps,        X_OM,
             -1.0/pitch,
@@ -1088,7 +1087,7 @@ void SimbodyPhysics::AddDynamicModelToSimbodySystem(
 
         // initialize stop stiffness and dissipation from joint parameters
         gzJoint->limitForce[0] =
-          Force::MobilityLinearStop(this->forces, mobod,
+          SimTK::Force::MobilityLinearStop(this->forces, mobod,
           SimTK::MobilizerQIndex(0), gzJoint->GetStopStiffness(0),
           gzJoint->GetStopDissipation(0), low, high);
 
@@ -1099,22 +1098,22 @@ void SimbodyPhysics::AddDynamicModelToSimbodySystem(
         // is zero.  This will allow user to change damping coefficients
         // on the fly.
         gzJoint->damper[0] =
-          Force::MobilityLinearDamper(this->forces, mobod, 0,
+          SimTK::Force::MobilityLinearDamper(this->forces, mobod, 0,
                                    gzJoint->GetDamping(0));
 
         // add spring (stiffness proportional to mass)
         gzJoint->spring[0] =
-          Force::MobilityLinearSpring(this->forces, mobod, 0,
+          SimTK::Force::MobilityLinearSpring(this->forces, mobod, 0,
             gzJoint->GetStiffness(0),
             gzJoint->GetSpringReferencePosition(0));
       }
       else if (type == "universal")
       {
-        UnitVec3 axis1(SimbodyPhysics::Vector3ToVec3(
+        SimTK::UnitVec3 axis1(SimbodyPhysics::Vector3ToVec3(
           gzJoint->AxisFrameOffset(0).RotateVector(
           gzJoint->LocalAxis(UniversalJoint<Joint>::AXIS_PARENT))));
         /// \TODO: check if this is right, or AxisFrameOffset(1) is needed.
-        UnitVec3 axis2(SimbodyPhysics::Vector3ToVec3(
+        SimTK::UnitVec3 axis2(SimbodyPhysics::Vector3ToVec3(
           gzJoint->AxisFrameOffset(0).RotateVector(
           gzJoint->LocalAxis(UniversalJoint<Joint>::AXIS_CHILD))));
 
@@ -1122,10 +1121,10 @@ void SimbodyPhysics::AddDynamicModelToSimbodySystem(
         // note X and Y are reversed because Simbody defines universal joint
         // rotation in body-fixed frames, whereas Gazebo/ODE uses space-fixed
         // frames.
-        Rotation R_JF(axis1, XAxis, axis2, YAxis);
-        Transform X_IF(X_IF0.R()*R_JF, X_IF0.p());
-        Transform X_OM(X_OM0.R()*R_JF, X_OM0.p());
-        MobilizedBody::Universal uJoint(
+        SimTK::Rotation R_JF(axis1, SimTK::XAxis, axis2, SimTK::YAxis);
+        SimTK::Transform X_IF(X_IF0.R()*R_JF, X_IF0.p());
+        SimTK::Transform X_OM(X_OM0.R()*R_JF, X_OM0.p());
+        SimTK::MobilizedBody::Universal uJoint(
             parentMobod,      X_IF,
             massProps,        X_OM,
             direction);
@@ -1138,7 +1137,7 @@ void SimbodyPhysics::AddDynamicModelToSimbodySystem(
 
           // initialize stop stiffness and dissipation from joint parameters
           gzJoint->limitForce[nj] =
-            Force::MobilityLinearStop(this->forces, mobod,
+            SimTK::Force::MobilityLinearStop(this->forces, mobod,
             SimTK::MobilizerQIndex(nj), gzJoint->GetStopStiffness(nj),
             gzJoint->GetStopDissipation(nj), low, high);
 
@@ -1154,11 +1153,11 @@ void SimbodyPhysics::AddDynamicModelToSimbodySystem(
           // is zero.  This will allow user to change damping coefficients
           // on the fly.
           gzJoint->damper[nj] =
-            Force::MobilityLinearDamper(this->forces, mobod, nj,
+            SimTK::Force::MobilityLinearDamper(this->forces, mobod, nj,
                                      gzJoint->GetDamping(nj));
           // add spring (stiffness proportional to mass)
           gzJoint->spring[nj] =
-            Force::MobilityLinearSpring(this->forces, mobod, nj,
+            SimTK::Force::MobilityLinearSpring(this->forces, mobod, nj,
               gzJoint->GetStiffness(nj),
               gzJoint->GetSpringReferencePosition(nj));
         }
@@ -1174,7 +1173,7 @@ void SimbodyPhysics::AddDynamicModelToSimbodySystem(
         // simbody always assumes axis is specified in the child link frame.
         // \TODO: come up with a test case where we might need to
         // flip transform based on isReversed flag.
-        UnitVec3 axis(
+        SimTK::UnitVec3 axis(
           SimbodyPhysics::Vector3ToVec3(
             gzJoint->AxisFrameOffset(0).RotateVector(
             gzJoint->LocalAxis(0))));
@@ -1185,10 +1184,10 @@ void SimbodyPhysics::AddDynamicModelToSimbodySystem(
         //          gzJoint->LocalAxis(0)) << "]\n";
 
         // Simbody's pin is along Z
-        Rotation R_JZ(axis, ZAxis);
-        Transform X_IF(X_IF0.R()*R_JZ, X_IF0.p());
-        Transform X_OM(X_OM0.R()*R_JZ, X_OM0.p());
-        MobilizedBody::Pin pinJoint(
+        SimTK::Rotation R_JZ(axis, SimTK::ZAxis);
+        SimTK::Transform X_IF(X_IF0.R()*R_JZ, X_IF0.p());
+        SimTK::Transform X_OM(X_OM0.R()*R_JZ, X_OM0.p());
+        SimTK::MobilizedBody::Pin pinJoint(
             parentMobod,      X_IF,
             massProps,              X_OM,
             direction);
@@ -1199,7 +1198,7 @@ void SimbodyPhysics::AddDynamicModelToSimbodySystem(
 
         // initialize stop stiffness and dissipation from joint parameters
         gzJoint->limitForce[0] =
-          Force::MobilityLinearStop(this->forces, mobod,
+          SimTK::Force::MobilityLinearStop(this->forces, mobod,
           SimTK::MobilizerQIndex(0), gzJoint->GetStopStiffness(0),
           gzJoint->GetStopDissipation(0), low, high);
 
@@ -1210,26 +1209,26 @@ void SimbodyPhysics::AddDynamicModelToSimbodySystem(
         // is zero.  This will allow user to change damping coefficients
         // on the fly.
         gzJoint->damper[0] =
-          Force::MobilityLinearDamper(this->forces, mobod, 0,
+          SimTK::Force::MobilityLinearDamper(this->forces, mobod, 0,
                                    gzJoint->GetDamping(0));
 
         // add spring (stiffness proportional to mass)
         gzJoint->spring[0] =
-          Force::MobilityLinearSpring(this->forces, mobod, 0,
+          SimTK::Force::MobilityLinearSpring(this->forces, mobod, 0,
             gzJoint->GetStiffness(0),
             gzJoint->GetSpringReferencePosition(0));
       }
       else if (type == "prismatic")
       {
-        UnitVec3 axis(SimbodyPhysics::Vector3ToVec3(
+        SimTK::UnitVec3 axis(SimbodyPhysics::Vector3ToVec3(
             gzJoint->AxisFrameOffset(0).RotateVector(
             gzJoint->LocalAxis(0))));
 
         // Simbody's slider is along X
-        Rotation R_JX(axis, XAxis);
-        Transform X_IF(X_IF0.R()*R_JX, X_IF0.p());
-        Transform X_OM(X_OM0.R()*R_JX, X_OM0.p());
-        MobilizedBody::Slider sliderJoint(
+        SimTK::Rotation R_JX(axis, SimTK::XAxis);
+        SimTK::Transform X_IF(X_IF0.R()*R_JX, X_IF0.p());
+        SimTK::Transform X_OM(X_OM0.R()*R_JX, X_OM0.p());
+        SimTK::MobilizedBody::Slider sliderJoint(
             parentMobod,      X_IF,
             massProps,              X_OM,
             direction);
@@ -1240,7 +1239,7 @@ void SimbodyPhysics::AddDynamicModelToSimbodySystem(
 
         // initialize stop stiffness and dissipation from joint parameters
         gzJoint->limitForce[0] =
-          Force::MobilityLinearStop(this->forces, mobod,
+          SimTK::Force::MobilityLinearStop(this->forces, mobod,
           SimTK::MobilizerQIndex(0), gzJoint->GetStopStiffness(0),
           gzJoint->GetStopDissipation(0), low, high);
 
@@ -1248,30 +1247,30 @@ void SimbodyPhysics::AddDynamicModelToSimbodySystem(
         // is zero.  This will allow user to change damping coefficients
         // on the fly.
         gzJoint->damper[0] =
-          Force::MobilityLinearDamper(this->forces, mobod, 0,
+          SimTK::Force::MobilityLinearDamper(this->forces, mobod, 0,
                                    gzJoint->GetDamping(0));
 
         // add spring (stiffness proportional to mass)
         gzJoint->spring[0] =
-          Force::MobilityLinearSpring(this->forces, mobod, 0,
+          SimTK::Force::MobilityLinearSpring(this->forces, mobod, 0,
             gzJoint->GetStiffness(0),
             gzJoint->GetSpringReferencePosition(0));
       }
       else if (type == "ball")
       {
-        MobilizedBody::Ball ballJoint(
+        SimTK::MobilizedBody::Ball ballJoint(
             parentMobod,  X_IF0,
             massProps,          X_OM0,
             direction);
-        Rotation defR_FM = isReversed
-            ? Rotation(~gzJoint->defxAB.R())
+        SimTK::Rotation defR_FM = isReversed
+            ? SimTK::Rotation(~gzJoint->defxAB.R())
             : gzJoint->defxAB.R();
         ballJoint.setDefaultRotation(defR_FM);
         mobod = ballJoint;
       }
       else if (type == "fixed")
       {
-        MobilizedBody::Weld fixedJoint(
+        SimTK::MobilizedBody::Weld fixedJoint(
             parentMobod,  X_IF0,
             massProps,    X_OM0);
         mobod = fixedJoint;
@@ -1311,7 +1310,7 @@ void SimbodyPhysics::AddDynamicModelToSimbodySystem(
       if (link->slaveMobods.empty()) continue;
       for (unsigned i = 0; i < link->slaveMobods.size(); ++i)
       {
-        Constraint::Weld weld(link->masterMobod, link->slaveMobods[i]);
+        SimTK::Constraint::Weld weld(link->masterMobod, link->slaveMobods[i]);
 
         // in case we want to know later
         link->slaveWelds.push_back(weld);
@@ -1322,7 +1321,7 @@ void SimbodyPhysics::AddDynamicModelToSimbodySystem(
   //   leave out optimization
   // // Add the loop joints if any.
   // for (int lcx=0; lcx < _mbgraph.getNumLoopConstraints(); ++lcx) {
-  //     const MultibodyGraphMaker::LoopConstraint& loop =
+  //     const SimTK::MultibodyGraphMaker::LoopConstraint& loop =
   //         _mbgraph.getLoopConstraint(lcx);
 
   //     SimbodyJointPtr joint(loop.getJointRef());
@@ -1330,18 +1329,18 @@ void SimbodyPhysics::AddDynamicModelToSimbodySystem(
   //     SimbodyLinkPtr  child(loop.getChildBodyRef());
 
   //     if (joint.type == "weld") {
-  //         Constraint::Weld weld(parent.masterMobod, joint.xPA,
+  //         SimTK::Constraint::Weld weld(parent.masterMobod, joint.xPA,
   //                               child.masterMobod,  joint.xCB);
   //         joint.constraint = weld;
   //     } else if (joint.type == "ball") {
-  //         Constraint::Ball ball(parent.masterMobod, joint.xPA.p(),
+  //         SimTK::Constraint::Ball ball(parent.masterMobod, joint.xPA.p(),
   //                               child.masterMobod,  joint.xCB.p());
   //         joint.constraint = ball;
   //     } else if (joint.type == "free") {
   //         // A "free" loop constraint is no constraint at all so we can
   //         // just ignore it. It might be more convenient if there were
-  //         // a 0-constraint Constraint::Free, just as there is a 0-mobility
-  //         // MobilizedBody::Weld.
+  //         // a 0-constraint SimTK::Constraint::Free, just as there is a 0-mobility
+  //         // SimTK::MobilizedBody::Weld.
   //     } else
   //         throw std::runtime_error(
   //             "Unrecognized loop constraint type '" + joint.type + "'.");
@@ -1403,7 +1402,7 @@ void SimbodyPhysics::SetSeed(uint32_t /*_seed*/)
 
 /////////////////////////////////////////////////
 void SimbodyPhysics::AddCollisionsToLink(const physics::SimbodyLink *_link,
-  MobilizedBody &_mobod, ContactCliqueId _modelClique)
+  SimTK::MobilizedBody &_mobod, SimTK::ContactCliqueId _modelClique)
 {
   // TODO: Edit physics::Surface class to support these properties
   // Define a material to use for contact. This is not very stiff.
@@ -1429,7 +1428,7 @@ void SimbodyPhysics::AddCollisionsToLink(const physics::SimbodyLink *_link,
   for (Collision_V::iterator ci =  collisions.begin();
                              ci !=  collisions.end(); ++ci)
   {
-    Transform X_LC =
+    SimTK::Transform X_LC =
       SimbodyPhysics::Pose2Transform((*ci)->RelativePose());
 
     // use pointer to store CollisionGeometry
@@ -1447,10 +1446,10 @@ void SimbodyPhysics::AddCollisionsToLink(const physics::SimbodyLink *_link,
         // rotate it based on normal vector specified by user
         // Create a rotation whos x-axis is in the
         // negative normal vector direction
-        Vec3 normal = SimbodyPhysics::Vector3ToVec3(p->Normal());
-        Rotation R_XN(-UnitVec3(normal), XAxis);
+        SimTK::Vec3 normal = SimbodyPhysics::Vector3ToVec3(p->Normal());
+        SimTK::Rotation R_XN(-SimTK::UnitVec3(normal), SimTK::XAxis);
 
-        ContactSurface surface(ContactGeometry::HalfSpace(), material);
+        SimTK::ContactSurface surface(SimTK::ContactGeometry::HalfSpace(), material);
 
         if (addModelClique)
             surface.joinClique(_modelClique);
@@ -1469,7 +1468,7 @@ void SimbodyPhysics::AddCollisionsToLink(const physics::SimbodyLink *_link,
         boost::shared_ptr<physics::SphereShape> s =
           boost::dynamic_pointer_cast<physics::SphereShape>((*ci)->GetShape());
         double r = s->GetRadius();
-        ContactSurface surface(ContactGeometry::Sphere(r), material);
+        SimTK::ContactSurface surface(SimTK::ContactGeometry::Sphere(r), material);
         if (addModelClique)
             surface.joinClique(_modelClique);
         int surfNum = _mobod.updBody().addContactSurface(X_LC, surface);
@@ -1491,13 +1490,13 @@ void SimbodyPhysics::AddCollisionsToLink(const physics::SimbodyLink *_link,
 
         // chunky hexagonal shape
         const int resolution = 1;
-        const PolygonalMesh mesh = PolygonalMesh::
-            createCylinderMesh(ZAxis, r, len/2, resolution);
-        const ContactGeometry::TriangleMesh triMesh(mesh);
-        ContactSurface surface(triMesh, material, 1 /*Thickness*/);
+        const SimTK::PolygonalMesh mesh = SimTK::PolygonalMesh::
+            createCylinderMesh(SimTK::ZAxis, r, len/2, resolution);
+        const SimTK::ContactGeometry::TriangleMesh triMesh(mesh);
+        SimTK::ContactSurface surface(triMesh, material, 1 /*Thickness*/);
 
-        // Vec3 esz = Vec3(r, r, len/2);  // Use ellipsoid instead
-        // ContactSurface surface(ContactGeometry::Ellipsoid(esz),
+        // SimTK::Vec3 esz = SimTK::Vec3(r, r, len/2);  // Use ellipsoid instead
+        // SimTK::ContactSurface surface(SimTK::ContactGeometry::Ellipsoid(esz),
         //                        material);
 
         if (addModelClique)
@@ -1513,7 +1512,7 @@ void SimbodyPhysics::AddCollisionsToLink(const physics::SimbodyLink *_link,
 
       case physics::Entity::BOX_SHAPE:
       {
-        Vec3 hsz = SimbodyPhysics::Vector3ToVec3(
+        SimTK::Vec3 hsz = SimbodyPhysics::Vector3ToVec3(
           (boost::dynamic_pointer_cast<physics::BoxShape>(
           (*ci)->GetShape()))->Size())/2;
 
@@ -1522,12 +1521,12 @@ void SimbodyPhysics::AddCollisionsToLink(const physics::SimbodyLink *_link,
         // number times to chop the longest side.
         const int resolution = 6;
         // const int resolution = 10 * (int)(max(hsz)/min(hsz) + 0.5);
-        const PolygonalMesh mesh = PolygonalMesh::
+        const SimTK::PolygonalMesh mesh = SimTK::PolygonalMesh::
             createBrickMesh(hsz, resolution);
-        const ContactGeometry::TriangleMesh triMesh(mesh);
-        ContactSurface surface(triMesh, material, 1 /*Thickness*/);
+        const SimTK::ContactGeometry::TriangleMesh triMesh(mesh);
+        SimTK::ContactSurface surface(triMesh, material, 1 /*Thickness*/);
 
-        // ContactSurface surface(ContactGeometry::Ellipsoid(hsz),
+        // SimTK::ContactSurface surface(SimTK::ContactGeometry::Ellipsoid(hsz),
         //                        material);
 
         if (addModelClique)

